mail_tracking/models/mail_tracking_email.py

```diff
@@ -163,16 +160,14 @@ def _compute_recipient_address(self):
             else:
                 email.recipient_address = email.recipient
 
-    @api.multi
     @api.depends('name', 'recipient')
-    def _compute_display_name(self):
+    def _compute_tracking_display_name(self):
```

Comment:
I renamed this function since it triggered a warning that the field depended on itself.
The _compute_display_name function in models.py adds display_name as a dependency, so to avoid this warning I had to rename the function...

My initial renaming didn't start with _compute. Travis is quite useful, we should consider adding it one day; would avoid new code with api.one's in it.